pdns/sdig.cc

```diff
@@ -249,7 +263,9 @@ try
           cerr<<"EDNS Subnet response: "<<reso.source.toString()<<", scope: "<<reso.scope.toString()<<", family = "<<reso.scope.getNetwork().sin4.sin_family<<endl;
 	}
       }
-
+      else if(iter->first == EDNSOptionCode::PADDING) {
+        cerr<<"EDNS Padding size: "<<(iter->second.size())<<endl;
```

Comment:
I think this `cerr` (and the ones before and after it) should be `cout`.

Reply:
pushed

Reply:
unpushed and saved at https://gist.github.com/Habbie/3719f3c68f7b0ced64c6228f8b60ddac because it involves updating a bunch of tests